### PR TITLE
Add dew point

### PR DESCRIPTION
**Pull request by piaskowyk**

### CODE REVIEW

File: src/device/Heater.cpp [hunk 1]
```diff
@@ -10,23 +17,26 @@ Heater::Heater() : communicationBus(config.oneWirePin) {
 
   thermometer = DallasTemperature(&communicationBus);
   thermometer.begin();
+  delay(2000);
+  thermometer.begin();
+  delay(1000);
```
Comment (jakubdybczak) [inline src/device/Heater.cpp:22]: I don't like this fragment, it should be investigated later imo - maybe at least add comment with information, why is this here?

File: src/device/Heater.cpp [hunk 2]
```diff
@@ -129,3 +202,25 @@ float Heater::getTemperature() {
...
+  }
+  return str;
+}
+
+float Heater::getHumidity() {
+  return weatherSensor.getHumidity();
+}
+
+float Heater::dewPoint(float humidity, float temperature) const {
+  return 243.04 * (log(humidity / 100.0) + ((17.625 * temperature) / (243.04 + temperature)))
```
Comment (jakubdybczak) [inline src/device/Heater.cpp:220]: If this is a well-known equation than there should be a comment with more informations (link to wikipedia?)

File: src/device/Heater.cpp [hunk 3]
```diff
@@ -49,65 +59,130 @@ Heater::~Heater() {
...
 void Heater::run() {
   heaterStatus.threadIsRunning = true;
   // xTaskCreatePinnedToCore(TaskFunction_t, Name, StackSize, void *pvParameters, Priority, TaskHandle_t, xCoreID)
-  xTaskCreatePinnedToCore(threadFunction, "termoThread", 10000, this, 1, &termoThreadHandler, 1);
+  xTaskCreatePinnedToCore(this->threadFunction, "termoThread", 10000, this, 1, &termoThreadHandler, 1);
   Logger::info("[Heater] run heater thread");
 }
 
-void threadFunction(void *pvParameters) {
+#ifdef CALCULATE_DEWPOIN
```
Comment (jakubdybczak) [inline src/device/Heater.cpp:86]: Typo: should be CALCULATE_DEWPOINT

File: src/device/Heater.cpp [hunk 4]
```diff
@@ -129,3 +202,25 @@ float Heater::getTemperature() {
...
+    str += String(deviceAddress[i], HEX);
+  }
+  return str;
+}
+
+float Heater::getHumidity() {
+  return weatherSensor.getHumidity();
+}
+
+float Heater::dewPoint(float humidity, float temperature) const {
```
Comment (jakubdybczak) [inline src/device/Heater.cpp:219]: Method name should have verb - for example calculateDewPoint

File: src/device/Heater.cpp (hunk 1 above)
Comment (piaskowyk) [reply src/device/Heater.cpp:22]: Me too bro, but I haven't idea why this happens. I'll add comment here and this issue should be debug in future.

File: src/device/Heater.cpp (hunk 2 above)
Comment (piaskowyk) [reply src/device/Heater.cpp:220]: Good point

**Pull request merged**